src/layouts/base-layout/index.tsx

```diff
@@ -4,7 +4,6 @@
  */
 
 // Third-party imports
-import { useRouter } from 'next/router'
```

Comment:
Excellent removal of the `pages` router — 1 less compatibility-thing to account for w/ AppRouter